### ISSUE TITLE
Vanilla Element Scanner does not detect elements registered via document.registerElement

### ISSUE THREAD

**Issue by MaKleSoft** [Polymer/polymer-analyzer#420]
Even though `document.registerElement` is deprecated, there are still some libraries that are using it (e.g. this one: https://github.com/github/time-elements) so I think it would be worthwhile to add support for it.

**Comment by MaKleSoft** [comment]
Huh, looking at the code it seems like another restriction of the current scanner is that elements need to be implemented via es6 class expressions to be detected. Thats probably something else that should be improved on.

**Comment by rictic** [comment]
Yeah, v0 element declarations are more likely to have more varied declaration syntax, but even trickier is getting any attribute information. The v1 `observedAttributes` property gives us a really useful hook for extracting the API of a custom element. For v0 elements I think we'd just have to rely on jsdoc comments.

**Comment by rictic** [comment]
We would accept a PR to add this though.
